project/npda/general_functions/index_multiple_deprivation.py

```diff
@@ -4,34 +4,50 @@
 
 # Standard imports
 import logging
-import requests
+import httpx
```

Comment:
The [fly.io blog post](https://fly.io/django-beats/running-tasks-concurrently-in-django-asynchronous-views/) suggested [httpx](https://www.python-httpx.org) and it looks great and is requests API compatible. I have no strong preference what we use though so long as it supports async and sadly httpx does bring [some transitive dependencies](https://www.python-httpx.org/#dependencies).